### ISSUE TITLE
improving ideas and planning

### ISSUE THREAD

**Issue by Trojan9** [hng-teamfury-org/mobile#3]

**Comment by Trojan9** [comment]
hello guys

**Comment by fridaychukwuamaka** [comment]
We need to divide the team:
To work on local storage for the history of the user and to store covid 19 prevention tip
To work on background code for notifications

**Comment by maroafenogho** [comment]
Thanks for creating this.
We need to have a demo. Do we have the designs yet?

**Comment by damisco87** [comment]
The design is still being worked on.

Kindly close the other issues created so that people can focus on this

**Comment by robincruseo** [comment]
> The design is still being worked on.
> 
> Kindly close the other issues created so that people can focus on this

ok

**Comment by damisco87** [comment]
Please, if you have been invited to join the Organization's Github, kindly join the mobile project

**Comment by damisco87** [comment]
Hi guys, this is the form to fill in your Github and slack details so that the team lead can add you to the Organization Github

https://docs.google.com/forms/d/e/1FAIpQLSd2slWRpRnGZiD0ZhHeckOSATUgF4VyA8Cky-iHOfbqpJTSRQ/viewform?usp=sf_link

**Comment by johnsani** [comment]
> Hi guys, this is the form to fill in your Github and slack details so that the team lead can add you to the Organization Github
> 
> https://docs.google.com/forms/d/e/1FAIpQLSd2slWRpRnGZiD0ZhHeckOSATUgF4VyA8Cky-iHOfbqpJTSRQ/viewform?usp=sf_link

Done

**Comment by damisco87** [comment]
while waiting for the design, I create this collaboration chart with other project teams, some have people working on them already so I have them as the people responsible and they will be giving us reports. Please let us identify other things we need to do before the design is ready. We need everyone now.
![collaboration chart](https://user-images.githubusercontent.com/50450408/84039054-98172c00-a998-11ea-95a4-815ae43e52a8.png)

**Comment by damisco87** [comment]
Any body here?

**Comment by maroafenogho** [comment]
Yeah

**Comment by damisco87** [comment]
Welcome Maro, The project leader is yet to be online. We need to move forward.

**Comment by maroafenogho** [comment]
We'll be needing reviews. Code written would need a review.

**Comment by maroafenogho** [comment]
@damisco. Yeah.

**Comment by damisco87** [comment]
Did you see the chat I posted above?
Is it possible to identify possible collaboration with other project teams?

**Comment by fridaychukwuamaka** [comment]
@damisco87 the coding team should review each other code

**Comment by maroafenogho** [comment]
Fair enough @fridaychukwuamaka 
We do know that the repo has been moved though right?

**Comment by damisco87** [comment]
Yeah, the coding team can begin to share ideas. Aside from coding, what other things?

The repo has been moved. The main team lead posted that on slack.

**Comment by johnsani** [comment]
What's the link for the new repo?

**Comment by damisco87** [comment]
When everyone joins the organization created, we will be able to see.

**Comment by maroafenogho** [comment]
Who are the flutter Devs we have and their proficiency?
I am a beginner there. I should be able to help with a few stuff though.

**Comment by robincruseo** [comment]
> Hi guys, this is the form to fill in your Github and slack details so that the team lead can add you to the Organization Github
> 
> https://docs.google.com/forms/d/e/1FAIpQLSd2slWRpRnGZiD0ZhHeckOSATUgF4VyA8Cky-iHOfbqpJTSRQ/viewform?usp=sf_link

done

**Comment by vectaaa** [comment]
Guys what are we doing pls...

**Comment by vectaaa** [comment]
I need some direction here

**Comment by damisco87** [comment]
@vectaaa Have you filled the Google form?

**Comment by robincruseo** [comment]
> Who are the flutter Devs we have and their proficiency?
> I am a beginner there. I should be able to help with a few stuff though.

same here

**Comment by johnsani** [comment]
Hi everyone, are we still waiting for the design team. Or the coding has started?

**Comment by maroafenogho** [comment]
Coding has started. @damisco87 and @fridaychukwuamaka are currently working on the WrokManager package to help manage notifications.

**Comment by maroafenogho** [comment]
The UI is going to be handled once we get the designs.

**Comment by vectaaa** [comment]
@damisco87 yes i have

**Comment by johnsani** [comment]
> The UI is going to be handled once we get the designs.

Alright Maro.

**Comment by uncleejay** [comment]
Any update?

**Comment by Trojan9** [comment]
@uncleejay send your slack and github username

**Comment by Trojan9** [comment]
please check your email for invites

**Comment by Trojan9** [comment]
everyone

**Comment by uncleejay** [comment]
@Trojan9 Github Username: Uncleejay 
Slack username : @Joseph
